discovery-provider/src/tasks/index_solana_plays.py

```diff
@@ -93,7 +95,6 @@ def get_latest_slot(db):
         f"index_solana_plays.py | returning {latest_slot} for highest slot")
     return latest_slot
 
-
```

Comment:
nit: the linter asks for new lines here, maybe we can keep them and the other places in this file